### ISSUE TITLE
Cannot install with: yarn install --ignore-optional

### ISSUE THREAD

**Issue by roblu** [techno-express/node-7z#2]
If I do that in my project that uses "node-7z-forall": `yarn install --ignore-optional`
I get:

```
Directory: C:\myproject\node_modules\node-7z-forall
Output:
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'all-unpacker'
```

It seems the dependency "all-unpacker" should not be an optional dependency, because it's always used in installer.js

**Comment by roblu** [comment]
It seems "retrying-promise" and "node-wget" should also not be optional.

**Comment by techno-express** [comment]
I see, i don't use yarn on a regular, will be fixing will updating soon.

**Comment by techno-express** [comment]
fixed now at version 1.0.2
